routes/project/kubernetes_cluster.rb

```diff
@@ -3,6 +3,8 @@
 class Clover
   hash_branch(:project_prefix, "kubernetes-cluster") do |r|
     r.web do
+      next unless @project.get_ff_kubernetes
```

Comment:
This check is missing in `routes/project/location/kubernetes_cluster.rb`.  Shouldn't it be there as well?

Reply:
I left it untouched on purpose, thinking that, if for some reason, someone is feature-blocked but has a provisioned cluster anyway, they should have some kind of access to that resource. I couldn't think of any security implications of it but if you think there is, we can add the check to location too.

What do you think?